packages/sil-voting-app/src/store/proposals/reducer.ts

```diff
@@ -1,34 +1,32 @@
+import { ChainId } from '@iov/bcp';
 import { Action } from 'redux';
 import { ActionType } from 'typesafe-actions';
 
 import { ProposalProps } from '../../routes/dashboard/components/Proposal';
 import * as actions from './actions';
 
 export interface SilProposal {
-  // Use ChainId type when governance API is integrated
-  readonly chainId: string;
+  readonly chainId: ChainId;
   readonly proposal: ProposalProps;
 }
 
+export type ProposalsState = SilProposal[];
```

Comment:
Removing the chain ID allows you to merge the types `SilProposal` and `ProposalProps`